### PR TITLE
readme: mention vs code extension

### PR DESCRIPTION
**Pull request by dberenbaum**
Adds a couple of brief references to the VS Code extension.

Questions/issues to consider:
* Where (else) in the readme should we mention the extension?
* Should we be giving any kind of summary or additional info about the extension in the DVC readme?
* The quickstart could be moved above "How DVC works", which would move up the reference to the extension. We should rewrite/reorganize the rest of the readme, but I didn't want to introduce unrelated changes in this PR.
* Studio might also be mentioned more and earlier (for example, when discussing sharing and comparing experiments), but this can be a separate discussion.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,8 +35,9 @@ learning projects:
 #. **Iterate** fast with straightforward YAML pipelines. When you make changes,
    only run the steps impacted by those changes.
 
-#. **Track** experiments in your local Git repo (no servers needed) and choose
-   which to keep and share.
+#. **Track** experiments in your local Git repo (no servers needed) and
+   [IDE](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc).
```
Comment (shcheklein) [inline README.rst:39]: IDE -> VS Code

File: README.rst [hunk 2]
```diff
@@ -95,6 +96,9 @@ Please read `Get Started <https://dvc.org/doc/get-started>`_ guide for a full ve
 |                                   | | ``$ dvc push``                                                           |
 +-----------------------------------+----------------------------------------------------------------------------+
 
+Prefer to work in your IDE? Check out the [DVC Extension for Visual Studio
```
Comment (shcheklein) [inline README.rst:99]: doesn't look very natural tbh

**Review by shcheklein — CHANGES_REQUESTED** [review]
A few ideas:
- add badge (instead of donate - we don't need it and it's not very useful anyway)
- add link (section before badges)
- add a link in the line 26
- even more extreme add "in CLI (or VS code)" to the line 17
- mention in the installation section

File: README.rst (hunk 2 above)
Comment (shcheklein) [reply README.rst:99]: sorry, I meant that the whole paragraph feels a bit too much here - looks like an ad a bit. If we have enough organic mentions, I would say we can completely skip it, wdyt?

File: README.rst [hunk 3]
```diff
@@ -7,8 +7,9 @@
 • `Chat (Community & Support) <https://dvc.org/chat>`_
 • `Tutorial <https://dvc.org/doc/get-started>`_
 • `Mailing List <https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&id=24c0ecc49a>`_
+• `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_
 
-|CI| |Maintainability| |Coverage| |Donate| |DOI|
+|CI| |Maintainability| |Coverage| |Donate| |VSCode| |DOI|
```
Comment (shcheklein) [inline README.rst:12]: VS Code

**Comment by shcheklein** [comment]
@dberenbaum let's merge it Monday end of the day? (there is a merge conflict, and could you please resolve the open conversations - just to make sure that you saw the feedback :) ).

File: README.rst (hunk 3 above)
Comment (dberenbaum) [reply README.rst]: This doesn't accept whitespace. It's not visible anyway on the page, so I don't think it's an issue. Let me know if you think it's a problem.

**Comment by dberenbaum** [comment]
Well I forgot this is rst and not md 🤦 . Please hold.

File: README.rst [hunk 4]
```diff
@@ -7,8 +7,9 @@
 • `Chat (Community & Support) <https://dvc.org/chat>`_
 • `Tutorial <https://dvc.org/doc/get-started>`_
 • `Mailing List <https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&id=24c0ecc49a>`_
+• `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_
 
-|CI| |Maintainability| |Coverage| |DOI|
+|CI| |Maintainability| |Coverage| |VSCode| |DOI|
```
Comment (shcheklein) [inline README.rst:12]: VS Code

Comment (shcheklein) [reply README.rst:12]: is this text visible?

Comment (shcheklein) [reply README.rst:12]: okay, probably not :)

Comment (dberenbaum) [reply README.rst:12]: @shcheklein It's not visible and won't work with whitespace. See https://github.com/iterative/dvc/pull/7859#discussion_r893955863

Comment (dberenbaum) [reply README.rst:12]: Source from https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations:

> Citation labels contain alphanumerics, underlines, hyphens and fullstop

Comment (dberenbaum) [reply README.rst]: Never mind, it does work per https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions, so I must have made some mistake when trying it before. Updated now.

**Comment by dberenbaum** [comment]
@shcheklein tldr changed to `VS Code` (I mistakenly thought rst couldn't handle whitespace there).

**Review by shcheklein — APPROVED** [review]

**Pull request merged**